a1/src/main/resources/db/migration/V1__Initial_schema.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS users(
```

Comment:
You have no foreign keys in your database model. Usually you would use foreign keys to ensure referential constraints between entities.